app/controllers/ratings_controller.rb

```diff
@@ -0,0 +1,19 @@
+class RatingsController < ApplicationController
```

Comment:
I think we can move this controller into `controllers/articles/ratings_controller.rb`

then we can use:
```ruby
expose :article
expose :rating
```